src/mysky.ts

```diff
@@ -22,6 +22,10 @@ const SEED_STORAGE_KEY = "seed";
 // Descriptive salt that should not be changed.
 const SALT_ENCRYPTED_PATH_SEED = "encrypted filesystem path seed";
 
+// SALT_MYSKY_ID_VERIFICATION is the prefix with which we salt the data that
+// MySky signs in order to be able to prove ownership of the MySky id.
+const SALT_MYSKY_ID_VERIFICATION = "MYSKY_ID_VERIFICATION";
```

Comment:
Should we update the `const` name, e.g. `SALT_MESSAGE_SIGNING`?

Reply:
(I think I get why it's named that way but it's kind of confusing.)

Reply:
Hm we could yeah, I think I named it this way because David started calling it "mysky id verification". I don't mind updating it to what you are proposing though. Will await the discussion in Discord and then make all necessary changes.